css/properties.json

```diff
@@ -2085,7 +2085,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/appearance"
   },
   "aspect-ratio": {
-    "syntax": "auto | <ratio>",
+    "syntax": "[ auto | <ratio> ]{1,2}",
```

Comment:
I don't think this syntax is quite right. It would allow `1/2 3/4`, but I think it should be:
```
auto | <ratio> | (auto <ratio>) | (<ratio> auto)
```

Reply:
Oh right, thanks!